cnf-certification-test/networking/netutil/netutil.go

```diff
@@ -81,24 +81,78 @@ func GetListeningPorts(cut *provider.Container) (map[PortInfo]bool, error) {
 const (
 	dumpNFTablesCmd       = "nft list ruleset"
 	dumpIPTablesCmd       = "iptables-save"
+	dumpIP6TablesCmd      = "ip6tables-save"
 	ipTablesLegacyWarning = "# Warning: iptables-legacy tables present, use iptables-legacy-save to see them"
+	// https://bugzilla.redhat.com/show_bug.cgi?id=1915027
+	openshiftMachineConfigNft = `table ip filter {
+		chain INPUT {
+			type filter hook input priority filter; policy accept;
+		}
+	
+		chain FORWARD {
+			type filter hook forward priority filter; policy accept;
+			meta l4proto tcp tcp dport 22623 tcp flags & (fin|syn|rst|ack) == syn counter packets 0 bytes 0 reject
+			meta l4proto tcp tcp dport 22624 tcp flags & (fin|syn|rst|ack) == syn counter packets 0 bytes 0 reject
+			meta l4proto tcp ip daddr 169.254.169.254 tcp dport != 53 counter packets 0 bytes 0 reject
+			meta l4proto udp ip daddr 169.254.169.254 udp dport 53 counter packets 0 bytes 0 reject
+		}
+	
+		chain OUTPUT {
+			type filter hook output priority filter; policy accept;
+			meta l4proto tcp tcp dport 22623 tcp flags & (fin|syn|rst|ack) == syn counter packets 0 bytes 0 reject
+			meta l4proto tcp tcp dport 22624 tcp flags & (fin|syn|rst|ack) == syn counter packets 0 bytes 0 reject
+			meta l4proto tcp ip daddr 169.254.169.254 tcp dport != 53 counter packets 0 bytes 0 reject
+			meta l4proto udp ip daddr 169.254.169.254 udp dport 53 counter packets 0 bytes 0 reject
+		}
+	}
+	`
+	// https://bugzilla.redhat.com/show_bug.cgi?id=1915027
+	openshiftMachineConfigIptables = `*filter
+	:INPUT ACCEPT [0:0]
+	:FORWARD ACCEPT [0:0]
+	:OUTPUT ACCEPT [0:0]
+	-A FORWARD -p tcp -m tcp --dport 22623 --tcp-flags FIN,SYN,RST,ACK SYN -j REJECT --reject-with icmp-port-unreachable
+	-A FORWARD -p tcp -m tcp --dport 22624 --tcp-flags FIN,SYN,RST,ACK SYN -j REJECT --reject-with icmp-port-unreachable
+	-A FORWARD -d 169.254.169.254/32 -p tcp -m tcp ! --dport 53 -j REJECT --reject-with icmp-port-unreachable
+	-A FORWARD -d 169.254.169.254/32 -p udp -m udp ! --dport 53 -j REJECT --reject-with icmp-port-unreachable
+	-A OUTPUT -p tcp -m tcp --dport 22623 --tcp-flags FIN,SYN,RST,ACK SYN -j REJECT --reject-with icmp-port-unreachable
+	-A OUTPUT -p tcp -m tcp --dport 22624 --tcp-flags FIN,SYN,RST,ACK SYN -j REJECT --reject-with icmp-port-unreachable
+	-A OUTPUT -d 169.254.169.254/32 -p tcp -m tcp ! --dport 53 -j REJECT --reject-with icmp-port-unreachable
+	-A OUTPUT -d 169.254.169.254/32 -p udp -m udp ! --dport 53 -j REJECT --reject-with icmp-port-unreachable
+	COMMIT
+	`
 )
 
-func isIPOrNSTablesPresent(cut *provider.Container, command string) (bool, error) {
+func stripSpaceTabLine(in string) string {
+	s1 := strings.ReplaceAll(in, "\n", "")
+	s2 := strings.ReplaceAll(s1, "\t", "")
+	return strings.ReplaceAll(s2, " ", "")
```

Comment:
replacer := strings.NewReplacer("\n", "", "\t", "", " ", "")
replacer.Replace(in)

Reply:
fixed

Reply:
There is no commit with the code change :-)